src/components/header/header.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, HStack, VStack } from "@nypl/design-system-react-components";
+import React, { useEffect, useState } from "react";
+import DCLogo from "../logo/logo";
+
+const Header = () => {
+  const [isScrolled, setIsScrolled] = useState(true);
+
+  useEffect(() => {
+    const handleScroll = () => {
+      if (window.scrollY > 100) {
+        setIsScrolled(false);
+      } else {
+        setIsScrolled(true);
+      }
+    };
+
+    window.addEventListener("scroll", handleScroll);
+
+    return () => {
+      window.removeEventListener("scroll", handleScroll);
+    };
+  }, []);
+  return (
+    <Box
+      position="sticky"
+      top={0}
+      py={{ sm: "xs", md: "s" }}
+      zIndex={10}
```

Comment:
Some DS components also have a z-index and the 10 value may be too low that other components will "bleed through". This is also a "magic number" but 9999 is big enough to work and always be on top of other elements on a page.

Reply:
9999 is actually so high it covers error messages from React so let me know if 999 is okay